src/main/java/br/com/zup/concessionariacitroen/controllers/DTOs/PedidoDTO.java

```diff
@@ -0,0 +1,80 @@
+package br.com.zup.concessionariacitroen.controllers.DTOs;
+
+import br.com.zup.concessionariacitroen.enums.TipoPagamento;
+import br.com.zup.concessionariacitroen.models.Automovel;
+import br.com.zup.concessionariacitroen.models.Cliente;
+import br.com.zup.concessionariacitroen.models.Pedido;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class PedidoDTO {
+    private int numeroPedido;
+    private Automovel automovel;
+    private Cliente cliente;
+    private LocalDate dataCompra;
+    private TipoPagamento tipoPagamento;
+
+    public PedidoDTO() {}
+
+    public PedidoDTO(int numeroPedido, Automovel automovel, Cliente cliente, LocalDate dataCompra, TipoPagamento tipoPagamento) {
+        this.numeroPedido = numeroPedido;
+        this.automovel = automovel;
+        this.cliente = cliente;
+        this.dataCompra = dataCompra;
+        this.tipoPagamento = tipoPagamento;
+    }
+
+    public int getNumeroPedido() {
+        return numeroPedido;
+    }
+
+    public void setNumeroPedido(int numeroPedido) {
+        this.numeroPedido = numeroPedido;
+    }
+
+    public Automovel getAutomovel() {
+        return automovel;
+    }
+
+    public void setAutomovel(Automovel automovel) {
+        this.automovel = automovel;
+    }
+
+    public Cliente getCliente() {
+        return cliente;
+    }
+
+    public void setCliente(Cliente cliente) {
+        this.cliente = cliente;
+    }
+
+    public LocalDate getDataCompra() {
+        return dataCompra;
+    }
+
+    public void setDataCompra(LocalDate dataCompra) {
+        this.dataCompra = dataCompra;
+    }
+
+    public TipoPagamento getTipoPagamento() {
+        return tipoPagamento;
+    }
+
+    public void setTipoPagamento(TipoPagamento tipoPagamento) {
+        this.tipoPagamento = tipoPagamento;
+    }
+
```

Comment:
Implementar o método converterDTOparaPedido
```suggestion
public Pedido converterDTOParaPedido(){
        Pedido pedido = new Pedido();
        pedido.setModelo(this.modelo);
        pedido.setTipoCarro(this.tipoCarro);
        pedido.setCor(this.cor);
        pedido.setMotor(this.motor);
        pedido.setTransmissao(this.transmissao);
        pedido.setRodas(this.rodas);

        return pedido;
    }
```